api4/apitestlib.go

```diff
@@ -83,6 +83,18 @@ func TearDown() {
 		teams := result.Data.([]*model.Team)
 
 		for _, t := range teams {
+			if cres := <-app.Srv.Store.Channel().SearchInTeam(t.Id, "fakechannel"); cres.Err != nil {
+				l4g.Error("Error tearing down test channel")
+			} else {
+				channels := cres.Data.(*model.ChannelList)
+
+				for _, c := range *channels {
+					if err := app.PermanentDeleteChannel(c); err != nil {
+						l4g.Error(err.Error())
+					}
+				}
+			}
```

Comment:
I think this is already covered by `app.PermanentDeleteTeam` below which deletes the channels for a team, unless there's a case I'm missing